config.go

```diff
@@ -727,6 +727,10 @@ func ValidateConfig(cfg Config, interceptor signal.Interceptor, fileParser,
 		return fmt.Errorf(funcName+": "+format, args...)
 	}
 	makeDirectory := func(dir string) error {
+		if directoryExists(dir) {
+			return nil
+		}
+
```

Comment:
I think `MkdirAll` will do nothing if the dir doesn't exist?

Reply:
Hmm... You are right. So I think https://github.com/lightningnetwork/lnd/issues/5927 is caused by incorrect permissions on the directory in the first place.

Reply:
I dropped the commit.

Reply:
Actually fixed the problem this time, see 875d6553.